lib/tasks/analytics.rake

```diff
@@ -0,0 +1,67 @@
+namespace :analytics do
+
+    desc "Cache Current Analytics"
```

Comment:
For some reason your indenting is 4 whitespace here (and just noticed it's like that in the admin rake file too). Definitely want to stick with 2 spaces.

Reply:
Looks like vscode is setting default spacing to 4 but only for a few files -- other files are defaulting to 2. Pushed a spacing fix